cql3/util.hh

```diff
@@ -81,6 +81,8 @@ sstring maybe_quote(const sstring& s);
 /// quote the identifier name in CQL, so that is what this function does does.
 sstring quote(const sstring& s);
 
+sstring single_quote(const sstring& s);
```

Comment:
added

Reply:
If it's for a literal, perhaps we should have a function that accepts a data_value() and therefore knows now to quote numbers, and knows not to search for single quotes in dates or IP address literals (a premature optimization, so let's not do it, but the API is more correct if it accepts a value instead of requiring the user to first convert the value to a string and then quote the string (or not) depending on the type.